include/liver.hrl

```diff
@@ -75,6 +76,7 @@
     not_positive_decimal    => <<"NOT_POSITIVE_DECIMAL">>,
     wrong_format            => <<"WRONG_FORMAT">>,
     wrong_date              => <<"WRONG_DATE">>,
+    wrong_time              => <<"WRONG_TIME">>,
```

Comment:
maybe `wrong_datetime`?

Reply:
date and time is validated seperately, so wrong_date for invalid date and wrong_time for invalid time